### ISSUE TITLE
"rofi-wayland" won't launch on official install

### ISSUE THREAD

**Issue by manfred3000** [Manjaro-Sway/manjaro-sway#154]
I've got `sway` installed alongside GNOME (still Manjaro) and I'm using my own configurations - so no `manjaro-sway-settings` installed. I just replaced `rofi-lbonn-wayland-git` and `sworkstyle` from the AUR with the packages from your repo.  The latter is working as expected, but `rofi-wayland` is giving me:
 ```
rofi -show drun                                                  10:45

(process:6045): X11Helper-WARNING **: 10:45:40.437: Failed to open display: (null)

(process:6045): Rofi-WARNING **: 10:45:40.437: Connection has error
```
The git version from the AUR is working fine.

I've got xwayland disabled, btw. Any ideas what might be wrong are much appreciated (or just close this, since I'm not running Manjaro sway edition)

Anyway, I'm checking the isos once in a while for "inspiration", and I think you've done a pretty good job!

**Comment by Mathew-D** [comment]
Thank you for bring this up, as it seems something was missed on the last package update for rofi to 1.7.  As soon as the maintain gets back I will have him look at the rofi-wayland package.

**Comment by Mathew-D** [comment]
If you try it now it should work correctly with rofi-wayland-1.7.0-3

**Comment by manfred3000** [comment]
> If you try it now it should work correctly with rofi-wayland-1.7.0-3

Unfortunately I'm still getting the same error message...
For what it's worth, here's my `config.rasi`:
```
configuration {

	modi: "drun,run";
//	font: "NotoSans 12";
	show-icons: true;
	icon-theme: "Papirus";
	sidebar-mode: true;
	disable-history: true;
		
}
```

**Comment by Mathew-D** [comment]
if you run rofi --help does it say xwayland?

**Comment by manfred3000** [comment]
> if you run rofi --help does it say xwayland?

No, it's just returning the exact same error as well :(

**Comment by Mathew-D** [comment]
Can you double check to make sure you are on 1.7.0-3.  As the PKGBUILD for it and the one from the AUR are basicly the same

**Comment by manfred3000** [comment]
When I build it manually using [this PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=rofi-lbonn-wayland-git), it's working without issues.

```
pacman -Qi rofi-wayland                                            21:54
Name                     : rofi-wayland
Version                  : 1.7.0-1
```

**Comment by Mathew-D** [comment]
If you check the [new PKGBuild](https://github.com/Manjaro-Sway/rofi-wayland/blob/master/PKGBUILD) we are using it is basicly the same as in 1.7.0-2 version of the package it was using the wrong branch and I copied the section from the AUR to fix the 1.7.0-3 package.

**Comment by Mathew-D** [comment]
Glad I was able to help.
